pkg/controller/podautoscaler/horizontal.go

```diff
@@ -39,6 +39,8 @@ const (
 	// TODO: make it a flag or HPA spec element.
 	tolerance = 0.1
 
+	defaultTargetCPUUtilizationPercentage = 80
```

Comment:
In general, changing default values is a breaking API change, so it doesn't make much sense to make the default value a knob.